engine/core/src/main/java/org/datacleaner/job/builder/AbstractComponentBuilder.java

```diff
@@ -322,11 +324,50 @@ public B setConfiguredProperty(String configuredName, Object value) {
     public B setConfiguredProperty(ConfiguredPropertyDescriptor configuredProperty, Object value) {
         final boolean changed = setConfiguredPropertyIfChanged(configuredProperty, value);
         if (changed) {
+            if (configuredProperty.isInputColumn()) {
+                for (TransformedInputColumn<?> transformedInputColumn : getTransformedInputColumns(value)) {
```

Comment:
Seems a little weird having non-functional for loop with a functional expression inside.